R/metadata.R

```diff
@@ -287,6 +287,7 @@ metadata <- function(constituent, flow, load.rate="", dates,
 #'   whether \code{new.metadata} is provided.
 #' @export
 updateMetadata <- function(metadata, new.metadata=NA, ..., validate=TRUE) {
+  old=NULL
```

Comment:
this is the only place in the entire file where `old` occurs. you shouldn't need to do this. could you please try building+checking again with this line deleted, and if you're still getting an error, copy-paste the full error here?